src/utils/index.js

```diff
@@ -9,14 +10,17 @@ var config      = require( '../config' ),
     log         = require( '../log' ),
     connStatus  = require( '../connection' );
 
+// Placeholder image to show when 404
+var placeholderImage = fs.readFileSync( './placeholders/404.jpg' );
+
 /*
  * write the error response headers
  *
  * @param     res           object        response object
  * @param     statusCode    integer       numeric response code to write
  */
-var writeErrorHead = function( res, statusCode ) {
-  res.writeHead( statusCode, {
+var writeHeadersWithStatusAndContentType = function( res, statusCode, contentType ) {
+  var headers = {
     'expires': '0',
```

Comment:
This should be capitalized to be consistent with the rest that we have listed below, and it should not be a zero value; it should be set to some value in the past. Remember to format the date correctly.